webrtc-c/canary/src/CloudwatchMonitoring.cpp

```diff
@@ -13,6 +13,9 @@ STATUS CloudwatchMonitoring::init()
     this->channelDimension.SetName("Channel");
     this->channelDimension.SetValue(pConfig->channelName.value);
 
+    this->labelDimension.SetName("Label");
```

Comment:
Just to be consistent, can we rename this to `WebRTCSDKCanaryType` or `WebRTCSDKCanaryLabel`?

Reply:
Updated to `WebRTCSDKCanaryLabel`